tests/framework/installer/ceph_installer.go

```diff
@@ -633,6 +633,11 @@ func (h *CephInstaller) UninstallRookFromMultipleNS(manifests ...CephManifests)
 	// Gather logs after status checks
 	h.GatherAllRookLogs(h.T().Name(), append([]string{h.settings.OperatorNamespace}, clusterNamespaces...)...)
 
+	// Gather pod restart count and alert
+	for _, namespace := range clusterNamespaces {
+		h.k8shelper.GetPodRestartsFromNamespace(namespace, h.T().Name(), utils.TestEnvName())
```

Comment:
How about just calling this line in the loop above  after line 630? Then no need for a separate loop.